src/common/com/intellij/plugins/haxe/ide/HaxeCompilerCompletionContributor.java

```diff
@@ -120,16 +123,24 @@ protected void addCompletions(@NotNull CompletionParameters parameters,
                        //Export/flash/haxe contains build.hxml which gets generated after build
                        break;
                      case HaxeModuleSettingsBaseImpl.USE_OPENFL:
-                       commandLineArguments.add(HaxelibCommandUtils.getHaxelibPath(moduleForFile));
-                       commandLineArguments.add("run");
-                       commandLineArguments.add("lime");
-                       commandLineArguments.add("display");
-                       //flash, html5, linux, etc
                        String targetFlag = moduleSettings.getNmeTarget().getTargetFlag();
-                       commandLineArguments.add(targetFlag);
 
-                       List<String> stdout = HaxelibCommandUtils.getProcessStdout(commandLineArguments,
-                                                                                  BuildProperties.getProjectBaseDir(project));
+                       List<String> stdout = openFLDisplayArguments.get(moduleForFile.getModuleFilePath() + targetFlag);
+                       if (stdout == null) {
```

Comment:
>I don't know enough about the compiler's completion yet. Doesn't it only give an answer for the code position you first asked for? If that's the case, and with this check, we'll always see the answer from the first request, and almost never get the answer for the current cursor position. Or am I missing something?

1. To have completion from Haxe compiler(`haxe --display {file}@{pos}` + parameters which is fast), it is used everywhere (Haxe, OpenFL/Lime, HXML). It requires compiler arguments(like `-src src -swf9 test.swf -debug` and etc, they are same as when you do it manually (Haxe/HXML) projects, where we manually pas this data to compiler.
2. For OpenFL's specific case, we don't have data like compiler arguments(OpenFL compiles it internally, when you do `openfl build flash`), it uses `neko` script, which embeds assets into SWF and calls Haxe with compiler arguments which are based on `project.xml` data and target